tests/suite/upgrade_test.go

```diff
@@ -111,6 +112,7 @@ var _ = Describe("Upgrade testing", Label("upgrade"), func() {
 			scheme   string
 		}
 		metricsCh := make(chan *metricsResults)
```

Comment:
This will need to be a buffered channel with size 2, otherwise one of the goroutines will block forever trying to write to the channel

Reply:
Yep, just ran into that block.